src/data/shared/header-menu.js

```diff
@@ -101,7 +101,6 @@ const headerMenu = [
       {
         title: 'Wikipedia',
         to: 'https://en.wikipedia.org/wiki/EBPF',
-        target: '_blank',
```

Comment:
from the deploy review, the link still is broken :/

Reply:
seems it is a configuration thing, let me ask PP